server/enumeration/authRole.ts

```diff
@@ -0,0 +1,10 @@
+enum AuthRole {
+  CASE_ADMIN = 'ROLE_LICENCE_CA',
+  RESPONSIBLE_OFFICER = 'ROLE_LICENCE_RO',
+  DECISION_MAKER = 'ROLE_LICENCE_DM',
+  READONLY = 'ROLE_LICENCE_READONLY',
+  ASSISTANT_CHIEF = 'ROLE_LICENCE_ACO',
+  SUPPORT = 'ROLE_NOMIS_BATCHLOAD',
```

Comment:
well just need CA, RO, DM and SUPPORT for now

Reply:
removed un wanted auth roles